library/src/styles/styles.css

```diff
@@ -0,0 +1,371 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .ai {
+  }
+  .ai__chevron {
+    @apply inline align-baseline cursor-pointer -mb-1;
+    width: 20px;
+    transform: rotate(-90deg);
+  }
+
+  .ai-info {
+    @apply text-left p-4;
+  }
+  .ai-info__title {
+    @apply text-3xl;
+  }
+  .ai-info__links {
+    @apply leading-normal mt-2 space-x-2 space-y-2;
+  }
+  .ai-info__links-item {
+    @apply inline-block;
+  }
+  .ai-info__links-item > a {
```

Comment:
If we're going to choose BEM over utility-first directly on the component, we should strictly avoid using CSS operators like `>`. They will become a problem if someone wants to override the styles applied here as our CSS selector will always have more specificity than theirs.

And honestly, I'd use `@apply` only for those highly repeated styles (like buttons, links, etc.). I'd keep the styles in the component files otherwise.

Reply:
> If we're going to choose BEM over utility-first directly on the component, we should strictly avoid using CSS operators like >. They will become a problem if someone wants to override the styles applied here as our CSS selector will always have more specificity than theirs.

I agree with it partially 😄 People will also be able to override styles by `ai-info__links-item > a`. Funny fact: Tailwind also uses operators in some classes, especially in `display` etc.

> And honestly, I'd use @apply only for those highly repeated styles (like buttons, links, etc.). I'd keep the styles in the component files otherwise.

And then people won't be able to override styles for particular parts :)

Reply:
> People will also be able to override styles by `ai-info__links-item > a`

Have you ever tried overriding Bootstrap styles? What do you think is the purpose of BEM? 😄 By doing this, you're breaking BEM and its benefits.

> And then people won't be able to override styles for particular parts :) I based styles on `@apply` directive having this problem in mind.

We don't want people to fully customize the components and all the CSS (unless they override CSS which is always possible). If someone wants to create a fully customized component, they'll be able to replace our components: https://github.com/fmvilas/asyncapi-pitches/blob/master/ui-library.md#customization. Not a problem to solve now.

In my experience, we should go the utility-first approach and maybe use some names for the colors, like primary, secondary, link, text, etc. Then people will be able to customize these colors. The same applies to sizes, paddings, margins, etc. It is, we must create a design system. We don't create CSS classes for the components but for the system the components are adhereing to.

Reply:
> We don't want people to fully customize the components and all the CSS (unless they override CSS which is always possible). If someone wants to create a fully customized component, they'll be able to replace our components: https://github.com/fmvilas/asyncapi-pitches/blob/master/ui-library.md#customization. Not a problem to solve now.

I go to link and read: `It should be easy to customize the components. Not just at CSS level but also at markup level.` 😄 I'm just kidding. 

@fmvilas I like the idea of ​​utility classes for buttons, borders, etc., and even today in the morning I noticed the problem that there are too many classes. The question is whether to start writing these classes, already or wait for the next cycle?

Reply:
I'd not move these classes to the CSS file in this cycle. It's a strong decision that will affect customization. In future cycles, we'll work on customization and I think it's then when we have to make a choice.

Reply:
I will remember! 😄 Style customization is most important thing after component unification. We talked about it under the main task of the cycle, but I found that not only Kyma has his own styles, but also people from Spotify or one person from https://github.com/asyncapi/asyncapi-react/issues/265 issue. The old component has css theming feature and we break it now.

Reply:
I don't like the styles. Not because of your changes but because the current design of the HTML template is broken and feels patched 😄 So yeah, I like the changes you did 😄